### PR TITLE
Expose Visualizer class

### PR DESCRIPTION
**Pull request by igor-davidyuk**
This PR introduces the Visualizer class to the user Python API to increase flexibility.

### CODE REVIEW

File: notebooks/003_upload_and_predict_image.ipynb [hunk 1]
```diff
@@ -207,15 +207,19 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from geti_sdk.utils import show_image_with_annotation_scene\n",
+    "import cv2\n",
+    "\n",
+    "from geti_sdk.prediction_visualization.visualizer import Visualizer\n",
```
Comment (ljcornel) [inline notebooks/003_upload_and_predict_image.ipynb:212]:
```suggestion
    "from geti_sdk.prediction_visualization import Visualizer\n",
```

File: notebooks/005_modify_image.ipynb [hunk 2]
```diff
@@ -219,8 +219,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "from geti_sdk.prediction_visualization.visualizer import Visualizer\n",
```
Comment (ljcornel) [inline notebooks/005_modify_image.ipynb:222]:
```suggestion
    "from geti_sdk.prediction_visualization import Visualizer\n",
```

File: notebooks/008_deploy_project.ipynb [hunk 3]
```diff
@@ -183,9 +185,33 @@
...
+    "t_start = time.time()\n",
+    "prediction_with_saliency_map = deployment.explain(numpy_rgb)\n",
+    "t_elapsed = time.time() - t_start\n",
+    "\n",
+    "print(\n",
+    "    f\"Running local inference with XAI on image took {t_elapsed*1000:.2f} milliseconds\"\n",
+    ")\n",
     "\n",
-    "show_image_with_annotation_scene(numpy_rgb, prediction, show_in_notebook=True);"
+    "result = visualizer.explain_label(\n",
```
Comment (ljcornel) [inline notebooks/008_deploy_project.ipynb:211]: Nice addition!

**Review by ljcornel — COMMENTED** [review]
Looks good! Few very minor comments, thanks for addressing this. The visualization API is much cleaner this way.

File: notebooks/003_upload_and_predict_image.ipynb (hunk 1 above)
Comment (igor-davidyuk) [reply notebooks/003_upload_and_predict_image.ipynb:212]: May be we go oposite direction and do
```python
from geti_sdk import Visualizer
```

File: notebooks/008_deploy_project.ipynb (hunk 3 above)
Comment (igor-davidyuk) [reply notebooks/008_deploy_project.ipynb:241]: I also added a markdown block with some explanation

File: geti_sdk/deployment/resources/OVMS_README.md [hunk 4]
```diff
@@ -62,8 +62,10 @@ image = cv2.cvtColor(image, cv2.COLOR_BGR2RGB)
 predictions = deployment.infer(image=image)
 
 # Show inference result
-from geti_sdk.utils import show_image_with_annotation_scene
-show_image_with_annotation_scene(image=image, annotation_scene=predictions);
+from geti_sdk.prediction_visualization.visualizer import Visualizer
```
Comment (ljcornel) [inline geti_sdk/deployment/resources/OVMS_README.md:65]:
```suggestion
from geti_sdk import Visualizer
```

File: notebooks/008_deploy_project.ipynb [hunk 5]
```diff
@@ -183,21 +185,65 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from geti_sdk.utils import show_image_with_annotation_scene\n",
+    "from geti_sdk.prediction_visualization.visualizer import Visualizer\n",
```
Comment (ljcornel) [inline notebooks/008_deploy_project.ipynb:188]:
```suggestion
    "from geti_sdk import Visualizer\n",
```

File: notebooks/010_model_serving.ipynb [hunk 6]
```diff
@@ -295,9 +295,11 @@
    },
    "outputs": [],
    "source": [
-    "from geti_sdk.utils import show_image_with_annotation_scene\n",
+    "from geti_sdk.prediction_visualization.visualizer import Visualizer\n",
```
Comment (ljcornel) [inline notebooks/010_model_serving.ipynb:298]:
```suggestion
    "from geti_sdk import Visualizer\n",
```

File: notebooks/012_post_inference_hooks.ipynb [hunk 7]
```diff
@@ -102,14 +102,16 @@
     "import cv2\n",
     "\n",
     "from geti_sdk.demos import EXAMPLE_IMAGE_PATH\n",
-    "from geti_sdk.utils import show_image_with_annotation_scene\n",
+    "from geti_sdk.prediction_visualization.visualizer import Visualizer\n",
```
Comment (ljcornel) [inline notebooks/012_post_inference_hooks.ipynb:105]:
```suggestion
    "from geti_sdk import Visualizer\n",
```

File: notebooks/use_cases/101_simulate_low_light_product_inspection.ipynb [hunk 8]
```diff
@@ -124,8 +124,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "from geti_sdk.prediction_visualization.visualizer import Visualizer\n",
```
Comment (ljcornel) [inline notebooks/use_cases/101_simulate_low_light_product_inspection.ipynb:127]:
```suggestion
    "from geti_sdk import Visualizer\n",
```

**Review by ljcornel — APPROVED** [review]

**Pull request merged**